src/main/java/io/github/sornerol/chess/pubapi/domain/game/ArchiveGame.java

```diff
@@ -41,4 +42,25 @@ public class ArchiveGame extends GameBase {
      */
     @JsonProperty("accuracies")
     private Accuracies accuracies;
+    
+    /**
+     * The Tcn
```

Comment:
I'm not familiar with "tcn." Is there any further explanation to what this value represents?

Reply:
I was just curious if there was a better explanation for what this value is. It's okay if we don't know right now. We can update the documentation at a later date.